lib/preview.coffee

```diff
@@ -12,6 +12,12 @@ module.exports =
     liveUpdate:
       type: 'boolean'
       default: true
+    glslangValidator:
```

Comment:
Would a different human-readable setting name convert better to title case?
`GLSLValidator -> GLSL Validator`